doc/release-notes.md

```diff
@@ -327,15 +330,13 @@ as described below.
 
 #### Creating Descriptor Wallets
 
-Descriptor Wallets are not created by default. They must be explicitly created using the
-`createwallet` RPC or via the GUI. A `descriptors` option has been added to `createwallet`.
-Setting `descriptors` to `true` will create a Descriptor Wallet instead of a Legacy Wallet.
+Descriptor wallets are not the default type of wallet.
 
 In the GUI, a checkbox has been added to the Create Wallet Dialog to indicate that a
-Descriptor Wallet should be created.
+Descriptor Wallet should be created. And a `descriptors` option has been added to `createwallet` RPC.
+Setting `descriptors` to `true` will create a Descriptor Wallet instead of a Legacy Wallet.
 
-Without those options being set, a Legacy Wallet will be created instead. Additionally the
-Default Wallet created upon first startup of Bitcoin Core will be a Legacy Wallet.
+Without those options being set, a Legacy Wallet will be created instead.
```

Comment:
Seems unrelated to the changes here. Probably best to put into https://github.com/bitcoin-core/bitcoin-devwiki/wiki/0.21.0-Release-Notes-Draft

Reply:
re: https://github.com/bitcoin/bitcoin/pull/20186#discussion_r507716803

> Seems unrelated to the changes here. Probably best to put into [bitcoin-core/bitcoin-devwiki/wiki/0.21.0-Release-Notes-Draft](https://github.com/bitcoin-core/bitcoin-devwiki/wiki/0.21.0-Release-Notes-Draft)

Thanks. Moved to wiki